custom/icds_reports/views.py

```diff
@@ -2448,3 +2448,49 @@ def get(self, request, *args, **kwargs):
     @property
     def bihar_state_id(self):
         return SQLLocation.objects.get(name='Bihar', location_type__name='state').location_id
+
+
+@location_safe
+@method_decorator([api_auth, toggles.ICDS_BIHAR_DEMOGRAPHICS_API.required_decorator()], name='dispatch')
```

Comment:
nit: may want to rename this flag if its used for more than the demographics api